coforma/swift-tech-challenge
# Issue/browse cards

# Title
Issue/Display College Tiles

# Summary
This PR creates mock college tiles on the browse page

Fixes #32

## Desktop
![Screenshot 2024-03-13 at 11 16 30 AM](https://github.com/coforma/swift-tech-challenge/assets/19439679/241528da-9ec5-4bf5-9b67-76b4dfe4c28e)

## Mobile
![Screenshot 2024-03-13 at 11 17 03 AM](https://github.com/coforma/swift-tech-challenge/assets/19439679/31f2363b-ee3b-46a2-9625-71657a9e0140)



## Type of change


# How To Test

1. run `yarn dev`, then navigate to `localhost:3000`
2. See the new mock cards in action!

# Checklist:
